app/admin.go

```diff
@@ -135,8 +215,53 @@ func (s *Server) GetLogsSkipSend(page, perPage int) ([]string, *model.AppError)
 	return lines, nil
 }
 
-func (a *App) GetLogsSkipSend(page, perPage int) ([]string, *model.AppError) {
-	return a.Srv().GetLogsSkipSend(page, perPage)
+func IsLogFilteredByLevel(logFilter *model.LogFilter, entry *model.LogEntry) bool {
```

Comment:
probably this should be private, and the next one too

Reply:
TIL that making it private means removing the first capital letter 🤣